io.openems.edge.batteryinverter.deye/src/io/openems/edge/deye/common/SurplusFeedInHandler.java

```diff
@@ -0,0 +1,212 @@
+package io.openems.edge.deye.common;
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.openems.common.exceptions.OpenemsError.OpenemsNamedException;
+import io.openems.common.utils.DoubleUtils;
+import io.openems.edge.common.channel.IntegerReadChannel;
+import io.openems.edge.common.channel.IntegerWriteChannel;
+import io.openems.edge.common.channel.StateChannel;
+import io.openems.edge.common.component.ComponentManager;
+import io.openems.edge.deye.common.charger.DeyeSunPv;
+import io.openems.edge.ess.dccharger.api.EssDcCharger;
+
+public class SurplusFeedInHandler {
+
+	private static final int GOING_DEACTIVATED_MINUTES = 15;
+	private static final float PV_LIMIT_FACTOR = 0.9f;
```

Comment:
All of these values are Commercial 40 related i guess..

Reply:
Removed.